### ISSUE TITLE
Using double-quotes in KV value (via wrangler)

### ISSUE THREAD

**Issue by WhitWaldo** [cloudflare/wrangler#1020]
#### Environment

* operating system: Windows 10 Pro
* output of `rustc -V`: rustc : The term 'rustc' is not recognized...
* output of `node -v`: v13.3.0
* output of `wrangler -V`: wrangler 1.6.0

#### Steps to reproduce
Goal: Attempt to store a value in KV store that contains a double quote (so I might parse it via JSON.parse upon retrieval):
```
wrangler kv:key put --binding=abc "test" "{"a":1}"
# error: Found argument 'a:1}' which wasn't expected, or isn't valid in this context
wrangler kv:key put --binding=abc "test" '{"a":1}'
# Yields {a:1}
wrangler kv:key put --binding=abc "test" '{""a"":1}'
# Yields {a:1}
wrangler kv:key put --binding=abc "test" '{`"a`":1}'
# Yields {`a`:1}
wrangler kv:key put --binding=abc "test" "{\"a\":1}"
# error: Found argument 'a\:1}' which wasn't expected, or isn't valid in this context
```

#### What did you expect to see?
If I manually set the same value in the browser via the new Workers editor, I can drop double quotes in there as I'd expect and retrieve in exactly the same fashion, as in:
`{"a":1}`

#### What did you see instead?
See above, examples following each attempt.

**Comment by gabbifish** [comment]
Hi there! Sorry you're running into this issue. A question: could you possibly upload things to Workers KV by using `'` or ``` for your outermost quotes? This should allow you to upload values with double quotes in them.

Alternatively, you can also use the `--path` argument to point to a file that contains `{"a":1}`, and upload the key-value pair that way.

**Comment by WhitWaldo** [comment]
Hello @gabbifish! 

I just gave it a shot with:

`wrangler kv:key put --binding=abc "test" `{"a":1}` `

...but my Powershell prompt doesn't seem to realize I'm at the end of the command and it's simply inserting a newline afterwards.

I've tried ' with the outermost quotes per the second example in my original post. When the key is queried back, it yields the string without the provided double quotes at all.
